### PR TITLE
[Doc] add 3.0.6 rn

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

File: docs/release_notes/release-3.0.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+- Data in partitions can be automatically cooled down over time. [#29335](https://github.com/StarRocks/starrocks/pull/29335) [#29393](https://github.com/StarRocks/starrocks/pull/29393)
+
+### Improvements
+
+- Supports implicit conversions for all compound predicates and for all expressions in the WHERE clause. You can enable or disable implicit conversions by using the [session variable](../reference/System_variable.md) `enable_strict_type`. The default value of this session variable is `false`.  [#21870](https://github.com/StarRocks/starrocks/pull/21870)
+- Unifies the logic between FE and BE in converting strings to integers. [#29969](https://github.com/StarRocks/starrocks/pull/29969)
+
+### Bug Fixes
+
+- If `enable_orc_late_materialization` is set to `true`, an unexpected result is returned when a Hive catalog is used to query struct-type data in ORC files. [#27971](https://github.com/StarRocks/starrocks/pull/27971)
```
Comment (amber-create) [inline docs/release_notes/release-3.0.md:19]: image
-> STRUCT-type

File: docs/release_notes/release-3.0.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+- During data queries through Hive Catalog, if a partitioning column and an OR operator are used in the WHERE clause, the query result is incorrect. [#28876](https://github.com/StarRocks/starrocks/pull/28876)
```
Comment (amber-create) [inline docs/release_notes/release-3.0.md:20]: image
"used" seems inappropriate. Suggest changing to "specified".

File: docs/release_notes/release-3.0.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+- The values returned by the RESTful API action `show_data` for cloud-native tables are incorrect. [#29473](https://github.com/StarRocks/starrocks/pull/29473)
+- If the [shared-data cluster](../deployment/deploy_shared_data.md) stores data in Azure Blob Storage and a table is created, the FE fails to start after the cluster is rolled back to version 3.0. [#29433](https://github.com/StarRocks/starrocks/pull/29433)
+- A user has no permission when querying a table in the Iceberg catalog even if the user is granted permission to that table. [#29173](https://github.com/StarRocks/starrocks/pull/29173)
```
Comment (amber-create) [inline docs/release_notes/release-3.0.md:23]: image
to -> on

File: docs/release_notes/release-3.0.md [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+- Online modifying the FE dynamic parameter `max_broker_load_job_concurrency` does not take effect.
+- The `Default` field values returned by the [SHOW FULL COLUMNS](../sql-reference/sql-statements/Administration/SHOW%20FULL%20COLUMNS.md) statement for columns of the [BITMAP](../sql-reference/sql-statements/data-types/BITMAP.md) or [HLL](../sql-reference/sql-statements/data-types/HLL.md) data type are incorrect. [#29510](https://github.com/StarRocks/starrocks/pull/29510)
+- The FE may fail to start when a materialized view is refreshed and at the same time, the refresh strategy of the materialized view is modified. [#29964](https://github.com/StarRocks/starrocks/pull/29964) [#29720](https://github.com/StarRocks/starrocks/pull/29720)
+- The error `unknown error` is returned when `select count(distinct(int+double)) from table_name` is executed. [#29691](https://github.com/StarRocks/starrocks/pull/29691)
+- After a primary key model table is restored, a metadata error issue happens when a BE is restarted, which leads to metadata inconsistency. [#30135](https://github.com/StarRocks/starrocks/pull/30135)
```
Comment (amber-create) [inline docs/release_notes/release-3.0.md:28]: image
-> a Primary Key table

Comment (amber-create) [inline docs/release_notes/release-3.0.md:28]: image
-> metadata errors occur and cause metadata inconsistencies occur if a BE is restarted?

File: docs/release_notes/release-3.0.md [hunk 5]
```diff
@@ -1,5 +1,32 @@
...
+- The values returned by the RESTful API action `show_data` for cloud-native tables are incorrect. [#29473](https://github.com/StarRocks/starrocks/pull/29473)
+- If the [shared-data cluster](../deployment/deploy_shared_data.md) stores data in Azure Blob Storage and a table is created, the FE fails to start after the cluster is rolled back to version 3.0. [#29433](https://github.com/StarRocks/starrocks/pull/29433)
+- A user has no permission when querying a table in the Iceberg catalog even if the user is granted permission on that table. [#29173](https://github.com/StarRocks/starrocks/pull/29173)
+- The `Default` field values returned by the [SHOW FULL COLUMNS](../sql-reference/sql-statements/Administration/SHOW%20FULL%20COLUMNS.md) statement for columns of the [BITMAP](../sql-reference/sql-statements/data-types/BITMAP.md) or [HLL](../sql-reference/sql-statements/data-types/HLL.md) data type are incorrect. [#29510](https://github.com/StarRocks/starrocks/pull/29510)
+- Online modifying the FE dynamic parameter `max_broker_load_job_concurrency` does not take effect.
```
Comment (evelynzhaojie) [inline docs/release_notes/release-3.0.md:25]: users may not understand the meaning of "online modifying" --- Modifying the FE dynamic parameter `max_broker_load_job_concurrency` using the ADMIN SET FRONTEND CONFIG command does not take effect

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
rerun backport-check

**Comment by hellolilyliuyi** [comment]
run behavior-check

**Comment by hellolilyliuyi** [comment]
@mergify backport  branch-3.2

**Pull request merged**